continuous-integration/linux/platform-dependent/fedora-35/setup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(realpath $(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd) )"
+
+dnf --assumeyes install \
+    bsdtar \
+    clinfo \
+    g++ \
+    libatomic \
+    python3-devel \
+    python3-pip \
+    wget \
+    || exit $?
+
+alternatives --install /usr/bin/python python /usr/bin/python3 0 || exit $?
+alternatives --install /usr/bin/pip pip /usr/bin/pip3 0 || exit $?
+
+source $(realpath $SCRIPT_DIR/../common.sh) || exit $?
+# Install OpenCL CPU runtime driver prerequisites
+dnf --assumeyes install \
+    numactl-libs \
+    redhat-lsb-core \
+    || exit $?
+install_opencl_cpu_runtime || exit $?
+
+function install_www_deb {
+    TMP_DIR=$(mktemp --tmpdir --directory zivid-python-install-www-deb-XXXX) || exit $?
+    pushd $TMP_DIR || exit $?
+    wget -nv "$@" || exit $?
```

Comment:
Always use the longer options if they exists.

For example
`-r` -> `--requirement`
`-y` -> `--assume-yes`
`-q` -> `--quiet`

Reply:
Same applies for some more instances below.

Reply:
Thx for the review. I agree in principle. However, this script is a verbatim copy of the Fedora 34 script (and probably 33 and 30 also). I do not think it is worth diverting the scripts for this.